### PR TITLE
Star refactor

### PR DESCRIPTION
**Pull request by zman811**
Changed to using the new stars component as well as made some small style changes

### CODE REVIEW

File: client/src/components/Reviews/components/ReviewList.jsx [hunk 1]
```diff
@@ -39,6 +44,12 @@ function ReviewList({reviewsMetadata, currentFilter }) {
       .catch((err) => console.log(err));
   }, [currentSort]);
   useEffect(() => {
+    if (!reviews.results) {
+      return;
+    }
+    if (reviews.results.length % 2 !== 0) {
```
Comment (slargman) [inline client/src/components/Reviews/components/ReviewList.jsx:50]: Is this to check if we've hit the end of the reviews?

**Review by slargman — COMMENTED** [review]
Good stuff. Looks ready to me.

File: client/src/components/Reviews/components/ReviewList.jsx (hunk 1 above)
Comment (zman811) [reply client/src/components/Reviews/components/ReviewList.jsx:50]: yeah it is

Comment (slargman) [reply client/src/components/Reviews/components/ReviewList.jsx:50]: Ah ok. How do you handle the case where the total number of reviews is even?

Comment (zman811) [reply client/src/components/Reviews/components/ReviewList.jsx:50]: ah good point i didnt think of that, ill go change that

File: client/src/components/Reviews/components/AddReview.jsx [hunk 2]
```diff
@@ -86,11 +81,12 @@ function AddReview({ handleModalToggle, allCharacteristics, productId }) {
     const submittedData = {
       ...reviewText,
       product_id: Number(productId),
-      rating,
+      rating: Number(rating),
       recommend: isRecommended,
       characteristics: characteristicSubmit,
       photos: [],
     };
+    console.log(submittedData);
```
Comment (slargman) [inline client/src/components/Reviews/components/AddReview.jsx:89]: Do you want to keep this `console.log` or is it for debugging?

**Review by slargman — APPROVED** [review]

File: client/src/components/Reviews/components/AddReview.jsx (hunk 2 above)
Comment (zman811) [reply client/src/components/Reviews/components/AddReview.jsx]: i forgot about it, ty

**Pull request merged**